03_intro_to_R_02_rstudio_cloud_tour.Rmd

```diff
@@ -6,13 +6,85 @@ output: html_document
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-# RStudio Cloud Tour
+# RStudio and Projects
```

Comment:
This information didn't come until much later originally, but I think it makes sense to move it up to when we are doing the RStudio Cloud Tour.